components/script/dom/element.rs

```diff
@@ -4143,23 +4146,6 @@ impl Element {
         }
     }
 
-    // https://html.spec.whatwg.org/multipage/#language
-    pub fn get_lang(&self) -> String {
-        self.upcast::<Node>()
-            .inclusive_ancestors(ShadowIncluding::Yes)
-            .filter_map(|node| {
-                node.downcast::<Element>().and_then(|el| {
-                    el.get_attribute(&ns!(xml), &local_name!("lang"))
-                        .or_else(|| el.get_attribute(&ns!(), &local_name!("lang")))
-                        .map(|attr| String::from(attr.Value()))
-                })
-                // TODO: Check meta tags for a pragma-set default language
-                // TODO: Check HTTP Content-Language header
-            })
-            .next()
-            .unwrap_or(String::new())
-    }
-
```

Comment:
Moved this to `Node` since any node inherits the `lang` attribute, not only elements. Although it's on elements that the attribute gets set.